src/native/containers/dn-vector-types.h

```diff
@@ -47,7 +47,7 @@ struct _ ## name ## _custom_params_t { \
 DN_DEFINE_VECTOR_T_STRUCT (dn_vector, uint8_t);
 
 typedef enum {
-	DN_VECTOR_ATTRIBUTE_DISABLE_MEMORY_INIT = 0x1
+	DN_VECTOR_ATTRIBUTE_MEMORY_INIT = 0x1
```

Comment:
We're changing the default?

Reply:
Yes I did, mainly since current EventPipe code uses g_array_new (FALSE, FALSE... and that memset showed up in profiling so figured that we should probably default to the more optimized alternative in new API. In Mono, there is no default for this, so all is decided on each callsite, meaning we could use dn_vector_t to replace g_array_new by using DN_VECTOR_ATTRIBUTE_MEMORY_INIT parameter.